src/store/model/amazon-de-warehouse.ts

```diff
@@ -0,0 +1,63 @@
+import {Store} from './store';
+
+export const AmazonDeWarehouse: Store = {
+	backoffStatusCodes: [403, 429, 503],
+	labels: {
+		captcha: {
+			container: 'body',
+			text: [
+				'geben sie die unten angezeigten zeichen ein',
+				'geben sie die zeichen unten ein'
+			]
+		},
+		inStock: {
+			container: '#a-autoid-0-announce',
+			text: ['In den Einkaufswagen']
+		},
+		maxPrice: {
+			container: 'a-size-large a-color-price olpOfferPrice a-text-bold',
+			euroFormat: false
+		},
+		outOfStock: [
+			{
+				container: '.a-size-medium',
+				text: [
+					'Derzeit gibt es keine Verkäufer, die diesen Artikel an Ihren Standort liefern können'
+				]
+			}
+		]
+	},
+	links: [
+		{
+			brand: 'test:brand',
+			model: 'test:model',
+			series: 'test:series',
+			url: 'https://www.amazon.de/gp/offer-listing/B087VM5XC6'
```

Comment:
Do you mind changing this to something that may not go out of stock for awhile?

Reply:
Of course, give me a few minutes